UWGD-Dimension-Doors/130-Project
# Show Victory scene with final score after beating SampleScene.

Testing Instructions:
- Start the game with SampleScene
- Beat the game (get to the little house)
- You should see the player sprite's thumbs up animation complete, and then you should be taken to a new scene with your score on it.
